README.md

```diff
@@ -1,892 +1,80 @@
 
-**Example return**
+But that's not all - you can actively participate in enhancing NFT security. Head over to our comprehensive tutorial on reporting NFT spam at [https://docs.moralis.io/web3-data-api/evm/report-nft-spam](https://docs.moralis.io/web3-data-api/evm/report-nft-spam).
 
-```json
-{
-  "status": "SYNCING",
-  "total": 2000,
-  "page": 2,
-  "page_size": 100,
-  "result": [
-    {
-      "token_address": "0x057Ec652A4F150f7FF94f089A38008f49a0DF88e",
-      "token_id": "15",
-      "contract_type": "ERC721",
-      "owner_of": "0x057Ec652A4F150f7FF94f089A38008f49a0DF88e",
-      "block_number": "88256",
-      "block_number_minted": "88256",
-      "token_uri": "string",
-      "metadata": "string",
-      "synced_at": "string",
-      "amount": "1",
-      "name": "CryptoKitties",
-      "symbol": "RARI"
-    }
-  ]
-}
-```
+This functionality is available on all Ethereum Virtual Machines (EVMs), but our initial classification specifically covers contracts on the Ethereum mainnet, Polygon mainnet, and Binance mainnet.
 
-### `GetWalletTokenIdTransfers`
+To identify which endpoints support spam detection, you can refer to the next section.
 
-NFT API gets an object with number of NFT transfers and an array with all transfers of NFT by token id (asynchronous).
+## List of NFT API Endpoints
 
-**Options**:
+The following table contains a list of all NFT API endpoints:
 
-- `chain` _(optional)_: The blockchain to get data from. Valid values are listed on the intro page in the [`Supported Blockchains`](#supported-blockchains). Default value Eth.
-- `format` _(optional)_: The format of the token id. Available values : decimal, hex. Default value : decimal.
-- `offset` _(optional)_: offset.
-- `limit` _(optional)_: limit.
-- `address` _(required)_: Address of the contract.
-- `token_id` _(requierd)_: The id of the token.
+| No. | Method                    | Description                                       | API Reference                                                                                             | URL                                                                       | Spam Detection |
+|-----|---------------------------|---------------------------------------------------|---------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------|----------------|
+| 1   | `getMultipleNFTs`        | Get multiple NFTs                                | [Method Documentation](https://docs.moralis.io/web3-data-api/evm/reference/get-multiple-nfts?tokens=[]&normalizeMetadata=false&media_items=true&chain=eth) | [https://deep-index.moralis.io/api/v2.2/nft/getMultipleNFTs](https://deep-index.moralis.io/api/v2.2/nft/getMultipleNFTs) | ✅             |
```

Comment:
For description maybe we can use description from our docs. 
Example: for getMultipleNFT's the description on our docs is Returns an array of NFTs specified in the request.